api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/model/IssueVulnerabilityDetails.java

```diff
@@ -45,4 +52,18 @@ public boolean areAllComponentVulnerabilitiesRemediated() {
         return vulnerabilitiesDeleted;
     }
 
+    public Optional<String> getHighestVulnerabilityAddedOrUpdated() {
```

Comment:
This method and the private method it calls should mention the severity in their name. e.g. `getHighestSeverityAddedOrUpdated`